pkg/api-server/testdata/policies_list.golden.json

```diff
@@ -0,0 +1,94 @@
+{
+ "policies": [
+  {
+   "name": "CircuitBreaker",
+   "readOnly": false,
+   "path": "circuit-breakers",
+   "displayName": "Circuit Breakers"
+  },
+  {
+   "name": "ExternalService",
+   "readOnly": false,
+   "path": "external-services",
+   "displayName": "External Services"
+  },
+  {
+   "name": "FaultInjection",
+   "readOnly": false,
+   "path": "fault-injections",
+   "displayName": "Fault Injections"
+  },
+  {
+   "name": "HealthCheck",
+   "readOnly": false,
+   "path": "health-checks",
+   "displayName": "Health Checks"
+  },
+  {
+   "name": "MeshGateway",
```

Comment:
I've said this before but we shouldn't call this a policy, it's required configuration for a type of `Dataplane`. If this is a policy, "policy" means nothing.

Reply:
I think both MeshGateway and ExternalServices are weird. I'm not sure it's the scope of this issue to fix this.

TBH I'm not sure what you define as a Policy. To me it's things you apply to a set of Dps to alter the conf. In this case MeshGateway fits this def

Reply:
Maybe I'm unsure. Feels like this is a team conversation to have here. In any case this is not the scope of this change (this only bases itself on the existing things we define as "a policy")

Reply:
I guess